signals/attachments/attachment_html_contains_redirecting.yml

```
@@ -0,0 +1,11 @@
+name: "Attachment: HTML contains fake redirects"
```

```suggestion
name: "Attachment: HTML contains 'redirect' in title block'"
```

or something like that. It doesn't seem like it's a fake redirect.